lib/ghomperm.gi

```diff
@@ -1684,6 +1717,23 @@ InstallMethod( ImagesSource, "blocks homomorphism",true,
   [ IsBlocksHomomorphism ], 0,
   RanImgSrcSurjBloho);
 
+#############################################################################
+##
+#M  KernelOfMultiplicativeGeneralMapping( <hom> ) . . . . . .  for blocks hom
+##
+InstallMethod( KernelOfMultiplicativeGeneralMapping,"blocks homomorphism",
+    true,
+    [ IsBlocksHomomorphism ], 0,
```

Comment:
So the first commit in this PR moves this method around (why), and also changes its rank to 200. The second commit changes the rank back to 0. Neither commit mentions any of this, though. What's going on?

Reply:
Also, the first and third commit kind of are invisible because the code they touch is completely rewritten by the other commits. All in all, perhaps this PR should be squashed into a single commit?

Reply:
The moving was to have the kernel method near the function that does the work. The ranking change was debugging code that should not have survived, and then was kicked out. I thought I had this commit already rebased away. Anyhow, I can squash it all together before merging.